stage1/stop_kvm/stop_kvm.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"syscall"
+
+	rktlog "github.com/coreos/rkt/pkg/log"
+	"github.com/coreos/rkt/stage1/common/ssh"
+)
+
+var (
+	force bool
+	log   *rktlog.Logger
+)
+
+func init() {
+	flag.BoolVar(&force, "force", false, "Forced stop")
+}
+
+func readIntFromFile(path string) (i int, err error) {
```

Comment:
> I don't have suggestions what to do

Let's put it in one of the _common_ packages, maybe `stage1/comon/...` in this case?